doc/architecture/domains/certificates.md

```diff
@@ -17,9 +17,7 @@ The reason for this split is certain constraints. The constraints are:
 
 ## Component diagram: Certificate API
 
-The component diagram shows a first iteration which is based on an in-memory integration event bus. A consequence of using the in-memory implementation is that all components that is dependent on the integration event bus must be in same container.
-
-Components that is used for mocking and will be replaced or discarded at a later are marked with its own color in the diagram.
+The component diagram shows how the solution works, based on a RabbitMQ message broker that publishes the events received from the DataSyncSyncer.
```

Comment:
In the code changes in the PR, the name "RabbitMQ" is used (domains/certificates/Query.API/API/RabbitMq/Startup.cs), but in the diagram it is called "Message Broker". Before it was called "Integration Event Bus" both in the code and in the diagram. Is it possible that we can have that the name of a component (or container) is named the same in both code and in the diagram?